pipeline_steps/multi_node_aio_prepare.groovy

```diff
@@ -107,10 +28,20 @@ def prepare() {
         git url: env.OSA_OPS_REPO, branch: "master"
         sh "git checkout ${env.OSA_OPS_BRANCH}"
       }
-      dir("openstack-ansible-ops/${env.MULTI_NODE_AIO_DIR}") {
+      dir("openstack-ansible-ops/multi-node-aio") {
+        sh """#!/bin/bash
+          # The multi-node-aio tool is quite modest when it comes to allocating
+          # RAM to VMs -- since we have RAM to spare we double that assigned to
+          # infra nodes.
+          # TODO(mattt): When https://review.openstack.org/#/c/507875/ merges
+          #              we can simply overwrite infra_vm_server_ram in group vars
+          sed -i 's/server_vm_ram: 8192/server_vm_ram: 16384/g' playbooks/host_vars/infra*.yml
+          cp -a ${WORKSPACE}/rpc-gating/scripts/dynamic_inventory.py playbooks/inventory
```

Comment:
this like can now be a simple environmental export:

``export MNAIO_INVENTORY="${WORKSPACE}/rpc-gating/scripts/dynamic_inventory.py"``

Reply:
Cool!  The dynamic_inventory.py is just to _extend_ the static inventory if a jenkins job needs additional compute or volume hosts, so I think we'll still need to put it into playbooks/inventory right?